src/tiles/selection-tile.component.ts

```diff
@@ -82,6 +83,14 @@ export class SelectionTile {
 		SelectionTile.tileCount++;
 	}
 
+	@HostListener("keydown", ["$event"])
+	keyboardInput(event) {
+		if (event.key === "Enter" || event.key === "Spacebar" || event.key === " ") {
```

Comment:
We should probably use `event.keyCode` and `event.which` here too.

Although they are both deprecated, some older browsers will only support those fields.

`.key`
![image](https://user-images.githubusercontent.com/14989804/69452987-fea46780-0d30-11ea-8570-6ad53059ae10.png)


`.which`
![image](https://user-images.githubusercontent.com/14989804/69453012-0b28c000-0d31-11ea-999a-2c4f765b5dc9.png)


`.keyCode`
![image](https://user-images.githubusercontent.com/14989804/69453045-1a0f7280-0d31-11ea-8e4e-a9596bda5e53.png)

Reply:
Looks like `.key` is supported down to IE9 ... we only support back to IE11 ... we're also only concerned with the latest version of everything else, and Firefox ESR

Reply:
👍